packages/ui-components/styles/index.css

```diff
@@ -13,15 +13,16 @@
 @import './base.css' layer(utilities);
 @import './markdown.css' layer(utilities);
 
-/*
-  The default border color has changed to `currentColor` in Tailwind CSS v4,
-  so we've added these compatibility styles to make sure everything still
-  looks the same as it did with Tailwind CSS v3.
-
-  If we ever want to remove these styles, we need to add an explicit border
-  color utility to any element that depends on these defaults.
-*/
 @layer base {
+  /* See https://tailwindcss.com/docs/upgrade-guide#buttons-use-the-default-cursor */
+  button,
+  [role='button'] {
+    &:not(:disabled) {
+      cursor: pointer;
+    }
+  }
```

Comment:
Very much a nit, but I think this was better being below both `:root` and the `*` block?